paris/src/testDebug/java/com/airbnb/paris/proxies/ViewStyleBuilderTest.kt

```diff
@@ -49,6 +51,34 @@ class ViewStyleBuilderTest {
         )
     }
 
+    @Test
+    fun backgroundTint() {
```

Comment:
This would be the `backgroundTintRes` test. Once you change the parameter type of the proxy you can add tests for `@ColorInt` and `ColorStateList` as well.